docs/license-faq_en.md

```
@@ -4,160 +4,156 @@
 ## Other
 
 **Q13: Is there a possibility of further license changes in the future?**
 
-A13: While not currently planned, changes may occur if necessary to ensure project sustainability. Any changes will be announced with sufficient notice.
+A13: There are no plans at present, but changes may be made if necessary to ensure the project's sustainability. If there are changes, we will provide sufficient notice.
 
-**Q14: How can I ask additional questions about licensing?**
+**Q14: What should I do if I have more questions about the license?**
 
-A14: Please contact support@aituberkit.com. We will respond as promptly as possible.
+A14: Please contact support@aituberkit.com. We will respond as quickly as possible.
```

Markdownlint

Bare URL used

(MD034, no-bare-urls)